RustFxA/FxAWebViewController.swift

```diff
@@ -31,21 +31,13 @@ class FxAWebViewController: UIViewController, WKNavigationDelegate {
      - parameter deepLinkParams: URL args passed in from deep link that propagate to FxA web view
      */
     init(pageType: FxAPageType, profile: Profile, dismissalStyle: DismissType, deepLinkParams: FxALaunchParams?) {
-        self.viewModel = FxAWebViewModel(
-            pageType: pageType,
-            profile: profile,
-            deepLinkParams: deepLinkParams,
-            firefoxAccounts: RustFirefoxAccounts.shared,
-            leanPlumClient: LeanPlumClient.shared
-        )
+        self.viewModel = FxAWebViewModel(pageType: pageType, profile: profile, deepLinkParams: deepLinkParams)
```

Comment:
Notes: `profile` has fxa as a property, leanplum is used as a shared by-convention and isn't passed around.